packages/browserify/test/globalRef.spec.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-undef */
```

Comment:
Rather than entirely disabling the rule here, how about listing the globals actually used?
```suggestion
/* global checkGlobal, checkSelf, checkThis, checkWindow */
```

Reply:
that won't work, because they're still unused

Reply:
this is what made it work:

```js
// eslint-disable-next-line no-unused-vars
/* global checkThis: true, checkSelf: true, checkWindow: true, checkGlobal: true */
```